env0/resource_environment.go

```diff
@@ -156,6 +272,84 @@ func getUpdatePayload(d *schema.ResourceData) client.EnvironmentUpdate {
 	return payload
 }
 
+func getDeployPayload(d *schema.ResourceData) client.DeployRequest {
```

Comment:
where we are setting the deployed / creator user id?

Reply:
I didn't, maybe we should, lets talk about irl
(for the record, if the api doesn't force it I think its fine \ can be added later)